soc/espressif/esp32/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if (CONFIG_SOC_ESP32_APPCPU)
+if(CONFIG_SOC_ESP32_APPCPU)
```

Comment:
these would need to change then too i.e. `CONFIG_SOC_ESP32_CPUAPP` (use `sed` for global search and replace)